airbyte-integrations/connectors/source-google-ads/metadata.yaml

```diff
@@ -62,6 +62,15 @@ data:
       - display_keyword_performance_report
       - shopping_performance_report
       - display_topics_performance_report
+      - accessible_bidding_strategy
```

Comment:
why those streams should be in suggested list?

Reply:
changed